tests/test_metadata.py

```diff
@@ -0,0 +1,31 @@
+import os
+import logging
+import pytest
+
+from schematic.models.metadata import MetadataModel
+from schematic.schemas.generator import SchemaGenerator
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+@pytest.fixture
+def metadata_model(helpers):
+
+        mm = MetadataModel(
+            helpers.get_data_path("example.model.jsonld"),
+            "local"
+        )
+
+        yield mm
+
+
+class TestMetadataModel:
+
+    def test_get_component_requirements(self, metadata_model):
```

Comment:
This test can take `as_graph` equal to both `True` and `False` using the `@pytest.mark.parametrize` syntax.

Reply:
Used the above syntax to allow the same test `test_get_component_requirements()` to take two parameter values for `as_graph` argument. It takes both `True` and `False`.